db/migrate/20160214115800_inline_ems_id.rb

```diff
@@ -0,0 +1,23 @@
+class InlineEmsId < ActiveRecord::Migration
+  def self.up
+    add_column :containers, :ems_id, :bigint
+    add_column :container_definitions, :ems_id, :bigint
+
+    say_with_time("Inline ems_id in containers") do
+      Container.includes(:container_group).all.each do |container|
```

Comment:
Needs a stub to avoid hitting real models in migrations...if you are using associations, you will need to use a `:class_name` option as well on those.